sdk/modelsrepository/Azure.Iot.ModelsRepository/src/DtmiConventions.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Azure.Iot.ModelsRepository
+{
+    internal class DtmiConventions
+    {
+        public static bool IsDtmi(string dtmi) => !string.IsNullOrEmpty(dtmi) && new Regex(@"^dtmi:[A-Za-z](?:[A-Za-z0-9_]*[A-Za-z0-9])?(?::[A-Za-z](?:[A-Za-z0-9_]*[A-Za-z0-9])?)*;[1-9][0-9]{0,8}$").IsMatch(dtmi);
```

Comment:
We may need to make a list of tasks somewhere for things like this. The regex shouldn't be created every time.

Reply:
task added